locales/en-US/notes.properties

```diff
@@ -123,3 +123,74 @@ newNote=New Note
 makePlainText=Make Plain Text
 deleteNote=Delete Note
 insufficientStorage=Notes Storage Full
+
+
+#
+# The following strings are used in the mobile app
+
+# LOCALIZATION NOTE (welcomeTitle4): This string is the login panel of the mobile app
+welcomeTitle4=Notes by Firefox
+
+# LOCALIZATION NOTE (welcomeHeadline): This string is the login panel of the mobile app
+welcomeHeadline=A simple, secure notepad app that syncs with Firefox
+
+# LOCALIZATION NOTE (signIn): This string is in the button in the login panel of the mobile app
+signIn=Sign In
+
+# LOCALIZATION NOTE (usageHint): This string is the login panel of the mobile app
+usageHint=To use Notes, you’ll need a Firefox Account and the free Notes browser extension from Firefox Test Pilot.
+
+# LOCALIZATION NOTE (usageLearnMore): This string is a link to the documentation
+usageLearnMore=Learn more
+
+# LOCALIZATION NOTE (drawerBtnFeedback): This string is a button in the mobile app options drawer
+drawerBtnFeedback=Feedback
+
+# LOCALIZATION NOTE (drawerBtnLogOut): This string is a button in the mobile app options drawer
+drawerBtnLogOut=Log out
+
+# LOCALIZATION NOTE (toastOffline): This string shows up as an Android Toast notification
+toastOffline=You are offline.
+
+# LOCALIZATION NOTE (toastLoggedInAs): Android Toast, {email} is the user's Firefox Account email.
+toastLoggedInAs=Logged in as {email}
+
+# LOCALIZATION NOTE (toastLoggedInError): Android Toast, {err} is the error code.
```

Comment:
It would be great to add an example of error code. Also, "Android Toast" -> "Notification"